src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+import { PeopleFilters } from '../components/PeopleFilters';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { useSearchParams } from 'react-router-dom';
+import { getPreparedPeople } from '../utils/getPreparedPeople';
+
+export const PeoplePage = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasError, setHasError] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+  const [searchParams] = useSearchParams();
+
+  const preparedPeople = getPreparedPeople(people, searchParams);
+
+  useEffect(() => {
+    setHasError(false);
+    setIsLoading(true);
+
+    getPeople()
+      .then(data => {
+        const updatedPeople = data.map(person => ({
+          ...person,
+          mother: data.find(p => p.name === person.motherName),
+          father: data.find(p => p.name === person.fatherName),
```

Comment:
The logic for finding the mother and father in the `updatedPeople` array assumes that the `name` property is unique. Ensure that this assumption holds true, or consider using a more reliable identifier if available, such as an ID, to avoid potential mismatches or errors.